main-support.ts

```diff
@@ -1,12 +1,26 @@
 import * as path from 'path';
+import mergeImg = require('merge-img');
 
 const fs = require('fs');
+const hasher = require('crypto').createHash;
 
 import { FinalObject, ImageElement } from './src/app/components/common/final-object.interface';
 import { acceptableFiles } from './main-filenames';
 
 import { globals } from './main-globals';
 
+/**
+ * Hash a given file using it's file name and size
+ * md5(file.name + file.size)
+ * @param fileName  -- file name to hash
+ * @param fileSize  -- file size to hash
+ */
+export function hashFile(fileName: string, fileSize: number): string {
+  // make the magic happen!
+  let hash = hasher('md5').update(fileName + fileSize.toString()).digest('hex');
```

Comment:
```suggestion
  const hash = hasher('md5').update(fileName + fileSize.toString()).digest('hex');
```
Please consider using `const`